supplementary_style_guide/style_guidelines/lead-in-sentences.adoc

```diff
@@ -0,0 +1,11 @@
+[[lead-in-sentences]]
+= Lead-in sentences
+
+A lead-in sentenece is the text that directly follows a procedure or prerequiste heading in a task-based module. It is not the module abstract, which describes the goals of the user for the module.
+
+.Example lead-in sentences
+----
+To register your system to Satellite Server, complete the following steps:
```

Comment:
Maybe add the .Procedure heading above it to be super clear about what we're talking about?

```
Example lead-in sentences
----
.Procedure

To register your system to Satellite Server, complete the following steps:
 ----
```

Reply:
Also suggest moving this example down below the part that says to avoid them, so that they don't see the example and stop reading before we tell them to avoid it :)

Reply:
Actually, looking at the example again, we might want to come up with a different example. What is here now is something that I'd say probably doesn't add a lot of value. I know this was pulled from what was linked in the spreadsheet here [1], but I think it was probably more of the Prereq section that they meant.

@redhat-documentation/ccs-style-council Does anyone have any other good examples of when a lead-in sentence **under the `.Prerequisites` or `.Procedure` heading** provides important context?

[1] https://access.redhat.com/documentation/en-us/red_hat_satellite/6.6/html-single/installing_capsule_server/index#registering-to-satellite-server_capsule

Reply:
@yetanothertw Vlada, I believe that you had some input on this issue.

Reply:
I am having difficulty understanding why a lead-in sentence would be necessary at all. I spoke with Bob Johnson about this and he recommended that content should be rewritten to avoid the use of a lead-in sentence. In the example from the spreadsheet, I think that the instances of lead-in sentences could be moved to the module abstract. Sorry, if I have confused this matter.

Reply:
@joaedwar Again, I want Vlada to weigh in on this. We had what I thought was a significant, decision-making conversation about it in one of the style council meetings. I recall that I felt as you and Bob do, and that Vlada had some good reasons.

Reply:
Hi folks, 

Back when we were discussing this a few months ago, we knew this was going to be tricky to get everyone's agreement on. There's both pro's and con's to using lead-in sentences. The main thought behind this recommendation was that it could be used in situations where your procedure has a complex set of prerequisites and you want to emphasise that ALL prerequisites must be met before you're able to proceed. Like in this example here:  https://access.redhat.com/documentation/en-us/red_hat_satellite/6.6/html-single/installing_capsule_server/index#registering-to-satellite-server_capsule
where you have two different groups of prerequisites.  

It's not about important context - I agree that important context should live in the abstract section or your context description. It's more about ease of navigation through your procedure and having a point of reference, especially if you consider accessibility issues and assume that your audience is using a screen reader. 
Consider that by the time your audience reaches the .Procedure heading, because of the amount of information contained in the previous section (prerequisites), it's very easy to lose focus of what this procedure is supposed to achieve. So you're reiterating or restating the purpose with your next lead-in sentence:  "To register your system to Satellite Server, complete the following steps:".  If you're not using a screen reader, you still have to scroll up before you find your title to remind yourself why you're here. So it acts as a reminder, or as a point of reference or anchor of sorts. 
I hope this makes sense, I wish I knew how to explain it better, sorry :-)  

I completely agree with @joaedwar that exceptions to the rule can be confusing, and that's not the intention here. Just want to make sure that in situations where it aids explanation and user navigation, lead-in sentences can and should be used. That also means they should be used scarcely, because most cases would be much simpler and more straightforward.

Reply:
Sorry to be late to the party. I have been swamped and am slowing digging out of my hole (how's that for some mixed metaphors?). But what Vlada is saying is exactly correct. We had a discussion of this and all agreed that the lead-in sentence is generally just fluff. Sometimes, it's good to have one (as in the situation that Vlada linked to), but generally just don't.

Also, if we do have a lead-in sentence, it should be a complete sentence.

Reply:
Thanks for that explanation @yetanothertw, that's really helpful.

I think what I'm struggling with now is how to display a good example. Or maybe we just don't. Maybe we could instead list a few of the examples that @yetanothertw mentioned.

What about something like the below for the entry? CC @redhat-documentation/ccs-style-council 

```
A lead-in sentence in this context is the text that directly follows a `Prerequisites` or `Procedure` heading in a task-based module. It is distinct from the module abstract, which describes the goals of the user for the module.

Do not use a lead-in sentence in the `Prerequisites` or `Procedure` sections of a module unless it is necessary to aid in navigation or add clarity.

The follow are examples where a lead-in sentence might add value:

* Your module has a very long list of prerequisites, and you want to remind users what task they are accomplishing when they get to the procedure.
* Your module has a complex set of prerequisites, and you want to emphasize that all prerequisites must be completed.

Use a complete sentence for the lead-in sentence, to reduce ambiguity and support translation.
```

Reply:
Looks good to me, @bergerhoffer  Thanks for putting that into words so eloquently!

Reply:
That's super helpful. Thanks for all the valuable input.